apps/condo/domains/common/components/TableFilter.tsx

```diff
@@ -1,6 +1,8 @@
 import React, { CSSProperties } from 'react'
 import isFunction from 'lodash/isFunction'
-import { Checkbox, Input, Space } from 'antd'
+import { Space } from 'antd'
+import Input from '@condo/domains/common/components/Input'
```

Comment:
Can we store wrapped ANTD components in a special folder to add CI check that you use the right component? like `@condo/domains/common/antd/Input` ?

Reply:
Done